apps/storefront/src/pages/UserManagement/getUserExtraFields.ts

```diff
@@ -66,7 +66,7 @@ const handleConversionExtraItemFormat = (userExtraFields: B2bExtraFieldsProps[])
         case 'number':
           currentItems.max = item.maximumValue || '';
           break;
-        case 'mutiline':
+        case 'multiline':
```

Comment:
might want to add `as const` to `FIELD_TYPE`:

```tsx
const FIELD_TYPE = {
  0: 'text',
  1: 'multiline',
  2: 'number',
  3: 'dropdown',
} as const;
```


<img width="931" alt="image" src="https://github.com/user-attachments/assets/358d1c31-5fe8-4b9b-9933-d27d32d1ace8" />

Reply:
done 👍 

- https://github.com/bigcommerce/b2b-buyer-portal/pull/335/commits/269e944705bd01ec02f9a4cdf2a559c3849a7f6c